docs/tutorials/training_tricks.md

```diff
@@ -50,3 +50,19 @@ model=dict(
 ```
 
 `class_weight` will be passed into `CrossEntropyLoss` as `weight` argument. Please refer to [PyTorch Doc](https://pytorch.org/docs/stable/nn.html?highlight=crossentropy#torch.nn.CrossEntropyLoss) for details.
+
+## Multiple Losses
+
+For loss calculation, we support multiple losses training concurrently. Here is an example config of training `unet` on `DRIVE` dataset, whose loss function is `1:3` weighted sum of `CrossEntropyLoss` and `DiceLoss`:
+
+```python
+_base_ = './fcn_unet_s5-d16_64x64_40k_drive.py'
+model = dict(
+    decode_head=dict(loss_decode=[dict(type='CrossEntropyLoss', name='CE', loss_weight=1.0),
+            dict(type='DiceLoss', name='Dice', loss_weight=3.0)]),
+    auxiliary_head=dict(loss_decode=[dict(type='CrossEntropyLoss', name='CE',loss_weight=1.0),
+            dict(type='DiceLoss', name='Dice', loss_weight=3.0)]),
+    )
```

Comment:
If the interface of mix loss has been changed, the doc should update accordingly (Chinese and English).